src/components/HostCalls/index.tsx

```diff
@@ -73,21 +82,39 @@ export const HostCalls = () => {
         }
       }}
     >
-      <DialogContent className="min-w-[80vw] min-h-[70vh]" hideClose={isOnEcalli}>
+      <DialogContent className="min-w-[80vw] min-h-[70vh] max-h-[70vh]" hideClose={isOnEcalli}>
         <div className="flex flex-col">
           <DialogHeader>
-            <DialogTitle className="mb-4">Storage</DialogTitle>
-            <DialogDescription>Please provide JSON storage or confirm empty</DialogDescription>
+            <DialogTitle className="mb-4">Define ecalli data</DialogTitle>
           </DialogHeader>
-          <div className="border-gray-500 border-2 rounded-lg h-full pt-1">
-            <TrieInput onChange={(v) => setNewStorage(v)} initialRows={storage} />
+          <div className="mt-3">
+            <span className="block text-md text-black font-bold mb-2">Service id</span>
```

Comment:
Service id should be a global parameter defined next to "batched steps" in the general settings dialog. Storage should just stay storage.